packages/instant-apps-components/src/components/instant-apps-keyboard-shortcuts/instant-apps-keyboard-shortcuts.scss

```diff
@@ -21,6 +21,10 @@
   table td, table th{
     border:solid 1px var(--calcite-ui-border-1);
     padding: 0.5rem 0.75rem 0.5rem 0.75rem;
+    white-space: normal;
+  }
+  
+  table td:lang(ja),table td:lang(ja-JP),table td:lang(ja-JP), table th:lang(ja){
```

Comment:
looks like you have the `table td:lang(ja-JP)` selector twice

Reply:
Thanks. Just replaced it with th